spec/02-integration/05-proxy/02-router_spec.lua

```diff
@@ -386,7 +388,7 @@ for _, strategy in helpers.each_strategy() do
       end)
 
       it("restricts a route to its 'hosts' if specified", function()
-        local ok, resp = proxy_client_grpc({
+        local ok, resp = assert(proxy_client_grpc({
```

Comment:
All of the below changes adding the assertion to the gRPC client calls should be part of a different commit to ensure atomicity of the git history (e.g. if we end up reverting the port matching commit for some reason in the future).

Reply:
removed.  they didn't change the test behaviour (there's an `assert.truthy(ok)` a few lines below), but result in much more information when failing, which helped a lot to see the real failure point